Backport change: bump driver version in order to distinguish between original
and new, backported, driver.
Index: src/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c
===================================================================
--- src.orig/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c	2017-04-11 15:31:52.966288046 +0200
+++ src/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c	2017-04-11 15:32:57.079699536 +0200
@@ -56,7 +56,7 @@
 static const char ixgbevf_driver_string[] =
 	"Intel(R) 10 Gigabit PCI Express Virtual Function Network Driver";
 
-#define DRV_VERSION "3.2.2-k-rh7.4"
+#define DRV_VERSION "3.2.2-k-rh7.4-z"
 const char ixgbevf_driver_version[] = DRV_VERSION;
 static char ixgbevf_copyright[] =
 	"Copyright (c) 2009 - 2015 Intel Corporation.";
